tasks/cpm_hardening.yml

```diff
@@ -48,11 +48,11 @@
   win_shell: |
     try
     {
-        $hardenRDS = if('{{ cpm_disable_import_inf }}' -eq 'true') {"True"} else {"False"}
+        $disbaleImportInf = if('{{ cpm_disable_import_inf }}' -eq 'true') {"True"} else {"False"}
         $filePath = "{{ cpm_installationautomation_folder }}\\CPM_Hardening_Config.xml"
         $xml = [xml](Get-Content $filePath)
-        $step1 = $xml.SelectSingleNode("//Step[@Name = 'RemoteDesktopServices']")
-        $step1.Parameters.Parameter.Value = $hardenRDS
+        $step1 = $xml.SelectSingleNode("//Step[@Name = 'ImportingINFConfiguration']")
+        $step1.Parameters.Parameter.Value = $disbaleImportInf
```

Comment:
Is there a logical issue? if we set disable import inf to true it will set disbaleImportInf to true which in turn will enable ImportingINFConfiguration.

Reply:
The naming here is confusing, if this step is set to true, it will update it's parameter that is called "IsPSMInstalled" which need to be updated to True